src/Client.php

```diff
@@ -564,11 +564,13 @@ public function upload($path)
         $guzzle = new GuzzleClient([
             'base_uri' => $this->getApiRoot()
         ]);
+        $fileinfo = pathinfo($path);
+        $filename = str_replace('.', '_', $fileinfo['filename']) . '.' . $fileinfo['extension'];
```

Comment:
What if we don't have extension? We will add `.` to the end of the name?

Reply:
You are right.. let me fix it